### DISCUSSION TITLE
polling doesnt start until a change is detected

### DISCUSSION THREAD

**Discussion by disarticulate** [maxpert/marmot#52] [Q&A]
I setup polling due to inability to detect changes (running inside docker; probably can fix this).

With two marmot services, one leader and one follower. Polling (scan_changes) only starts after an insert/update is performed

Expected behavior: polling/`scan_changes` should start on bootup

**Comment by maxpert** [comment]
Is the intention to propagate changes in the DB that were made while Marmot was not running? If yes then Marmot already does picks up and propagates changes at boot time.

**Comment by disarticulate** [comment]
I'm not seeing changes propagate on start up configuring the S3 system. Do both clients need to have separate snapshot locations? That's the only thing I can think of in the configuration. Right now, all I've observed are transmitted updates when two clients are connected. It does not appear that either client fetches a snapshot.

**Comment by maxpert** [comment]
No both clients need to point to same snapshot location. When Marmot boots up it looks at the sequence file to check if it's too behind in change log for sequence. If that file is missing, all sequences should be assumed 0. Try changing change log size to lower value to reproduce it.
